deployment/src/main/resources/dev-ui/qwc-neo4j-ogm-entities.js

```diff
@@ -0,0 +1,147 @@
+import { LitElement, html, css} from 'lit';
+import { observers } from 'build-time-data';
+import { columnBodyRenderer } from '@vaadin/grid/lit.js';
+import '@vaadin/grid';
+import '@vaadin/vertical-layout';
+import 'qui-badge';
+
+/**
+ * This component shows the Arc Observers
+ */
+export class QwcArcObservers extends LitElement {
```

Comment:
Is this used ? If so you probably want to rename this to `QwcNeo4JOgmEntities`

Reply:
Thanks, no, it isn't. I started by copy&pasta to learn the API and then found the table page builder, which is totally sufficient for my use case. Deleted the file.